symplyphysics/laws/astronomy/luminosity_of_Sun_in_future_from_luminosity_in_present.py

```diff
@@ -0,0 +1,42 @@
+from sympy import (Eq, solve, S,)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    convert_to,
+    dimensionless,
+)
+
+# Description
+## The luminosity of the Sun in the future is related to the luminosity of the Sun in the present. Luminosity is
+## indicated in units of solar luminosity. One unit is equal to the luminosity of the Sun at a given time.
```

Comment:
I don't think it is always true. Eventually luminosity will start reducing. Add conditions when this law is valid.

Reply:
> I don't think it is always true. Eventually luminosity will start reducing. Add conditions when this law is valid.

The essence of this formula is to link the luminosity in the future with the luminosity in the present. We are talking about the fact that, for example, at least in 100 million years, the luminosity of the sun, which will be considered luminosity at that time in the present time, will be less than the current luminosity? And then it will be impossible to use this formula?

Reply:
Look at this picture: https://faculty.wcas.northwestern.edu/infocom/The%20Website/end.html
So basically your formula is valid while Sun is on the main sequence.